front/src/modules/ui/navigation/mobile-navbar/tab-bar/components/TabBar.tsx

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { useRecoilState } from 'recoil';
+
+import { useCommandMenu } from '@/command-menu/hooks/useCommandMenu';
+import {
+  IconCheckbox,
+  IconList,
+  IconSearch,
+  IconSettings,
+} from '@/ui/display/icon';
+import { useIsMenuNavbarDisplayed } from '@/ui/layout/hooks/useIsMenuNavbarDisplayed';
+import { useIsSubMenuNavbarDisplayed } from '@/ui/layout/hooks/useIsSubMenuNavbarDisplayed';
+import { isNavbarOpenedState } from '@/ui/layout/states/isNavbarOpenedState';
+import { useIsMobile } from '@/ui/utilities/responsive/hooks/useIsMobile';
+
+const StyledContainer = styled.div<{
+  isMobile: boolean;
+}>`
+  bottom: ${({ theme }) => theme.spacing(4)};
+  display: ${({ isMobile }) => (isMobile ? 'flex' : 'none')};
+  gap: ${({ theme }) => theme.spacing(4)};
+  justify-content: center;
+  position: fixed;
+  width: 100%;
+`;
+const StyledIconContainer = styled.div<{ isActive?: boolean }>`
+  align-items: center;
+  aspect-ratio: 1/1;
+  background-color: ${({ isActive, theme }) =>
+    isActive ? theme.background.tertiary : theme.background.transparent};
+  border-radius: ${({ theme }) => theme.spacing(1)};
+  display: flex;
+  justify-content: center;
+  width: ${({ theme }) => theme.icon.size.xl}px;
+`;
+
+type IconT = 'tab' | 'search' | 'tasks' | 'settings';
+
+const TabBar = () => {
+  const currentPath = useLocation().pathname;
+  const navigate = useNavigate();
+  const [isNavbarOpened, setIsNavbarOpened] =
```

Comment:
'isNavbarOpened' is assigned a value but never used. Allowed unused vars must match /^_/u